templates/web/base/dashboard/index.html

```diff
@@ -159,12 +159,12 @@ <h1>[% loc('Summary statistics') %]</h1>
     [% IF category_to_group.$k_safe AND category_to_group.$k_safe != current_category %]
     [% SET current_category = category_to_group.$k_safe %]
       <tr>
-        <th scope="row">[% category_to_group.$k_safe %]</th>
-        <th scope="row">-</th>
+        <th colspan="(2 or 5)" scope="colgroup">[% category_to_group.$k_safe %]</th>
```

Comment:
Sorry I wasn't clear enough here - by `2 or 5` I meant you had to put 5 if group_by was category+state (because it has 3 extra columns of output in the table), or 2 otherwise - it's the number of columns that are spanned. And by "only need the one entry" I meant then you wouldn't have any of the `<th>-</th>` because this column is spanning them.